firmware/integration/rusefi_config.txt

```diff
@@ -373,7 +373,7 @@ bit launchControlEnabled;
 bit rollingLaunchEnabled;
 bit antiLagEnabled;
 bit useRunningMathForCranking,Fuel Map,Fixed;
-bit issue_294_25;
+bit displayLogicLevelsInEngineSniffer;
```

Comment:
it looks like you have a merge conflict - can you rebase on master?

Reply:
will do